src/main.c

```diff
@@ -0,0 +1,118 @@
+#include "merc.h"
+
+#include "doorman/doorman.h"
+
+#include <signal.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/time.h>
+#include <unistd.h>
+
+/* SIGTRAP on/off */
+bool debug = FALSE;
+/* check the mobs/objects */
+bool printinfo = FALSE;
```

Comment:
Ooooh...I'd forgotten about this. It calls report_object(),  report_mobile().  This could be very useful for balancing efforts.

Reply:
So much crupht but yeah